src/main/java/sim/coffee/SharedQueue.java

```diff
@@ -20,8 +27,18 @@
 		}
 
 		Order[] customerOrder = queue.getFirst();
-		log.add(customerOrder, Logger.OrderState.EXIT);
+		
+		switch (queueType) {
+		case CUSTOMER:
```

Comment:
This follows on from my comment on the Logger, we probably shouldn't have the SharedQueue be so hardcoded and specifically aware of certain use cases (considering it from an object oriented point of view). By changing the logger it also allows this class to become more cohesive.